internal/signature/sign.go

```diff
@@ -60,61 +68,70 @@ type Identity interface {
 
 // Sign signs a given payload for the given identity.
 // The resulting signature and cert used is returned.
-func Sign(ident Identity, body []byte, opts SignOptions) ([]byte, *x509.Certificate, error) {
+func Sign(ctx context.Context, ident Identity, body []byte, opts SignOptions) ([]byte, *x509.Certificate, *models.LogEntryAnon, error) {
```

Comment:
Idle thought: maybe worth making a struct to wrap the sig+cert+log entry?

Reply:
Done!